Expand Zod validation to include `fallback` w/ optional inputs & fix `AbiError` validation (#65)
wagmi-dev/abitype: .changeset/dull-taxis-share.md

---
'abitype': patch
---

> - "fallback" type supports optional inputs (supports legacy contracts like WETH)
> - fix AbiError "type": change from z.literal("event") to z.literal("error")

Reviewer: ```suggestion
Fixed `abitype/zod` support for legacy contracts, like WETH, by making `inputs` optional for `type: 'fallback'` and `AbiError` `type` from `"event"` to `"error"`.
```
